pkg/view/component/table.go

```diff
@@ -27,6 +28,43 @@ type TableConfig struct {
 	EmptyContent string                 `json:"emptyContent"`
 	Loading      bool                   `json:"loading"`
 	Filters      map[string]TableFilter `json:"filters"`
+	ButtonGroup  *ButtonGroup           `json:"buttonGroup,omitempty"`
```

Comment:
For consistency, can we also have a method like:

```
func (t *Table) AddButtonGroup(buttonGroup *ButtonGroup) {
	t.Config.ButtonGroup = buttonGroup
}
```

Reply:
Added an `AddButton` method instead. It will append to an existing button group or create a new one if nil.